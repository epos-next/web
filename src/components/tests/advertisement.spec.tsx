import React from "react"
import Advertisement from "@components/advertisement";
import { render } from "@testing-library/react";
import "jest-styled-components"

it("should match snapshot", () => {
    const { asFragment } = render(<Advertisement/>)
    expect(asFragment()).toMatchSnapshot()
});
