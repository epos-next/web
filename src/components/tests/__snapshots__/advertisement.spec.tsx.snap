// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  .c2 {
  font-size: 15px;
  line-height: 18px;
  color: #696969;
}

.c1 {
  width: 7px;
  height: 7px;
  background-color: var(--light-primary);
  border-radius: 50%;
  margin-right: 10px;
  margin-top: 5px;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
    class="c0 advertisement-component"
  >
    <div
      class="c1"
    />
    <p
      class="c2"
    />
  </div>
</DocumentFragment>
`;
